pkg/webui/console/store/middleware/logics/init.js

```diff
@@ -42,7 +42,11 @@ const consoleAppLogic = createRequestLogic({
       try {
         const userId = info.data.oauth_access_token.user_ids.user_id
         const result = await api.users.get(userId)
-        dispatch(user.getUserMeSuccess(result.data))
+        if (result.code === 7) {
```

Comment:
Shouldn't you be looking at HTTP response codes?

Reply:
Yep, indeed.